src/libraries/Microsoft.PowerFx.Interpreter/PrettyPrint.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    public class PrettyPrint
+    {
+        public static string GetPrettyPrint(string input)

```

Comment:
>GetPrettyPrint [](http://example.com/codeflow?start=29&length=14)

We need to have a better name for this. Calling `PrettyPrint.GetPrettyPrint` does not follow the rest of the project.